_stories/molecules/SearchMultiSelect/index.js

```diff
@@ -12,6 +12,14 @@ class TestSearchMultiSelect extends React.Component {
         names: namesList
     };
 
+    componentDidMount() {
```

Comment:
Just a thought, we could also add a knob button to set focus when clicked.

Reply:
Done